docs/dscom.build/TlbExport.sequence.puml

```diff
@@ -0,0 +1,64 @@
+@startuml "TlbExport"

```

Comment:
I would not use puml files. I believe that no one maintains that

Reply:
I do use puml files and I would keep it, if it is ok for you.

The file has been moved to /src/dsom.build/doc in e36ea6731206b710e736dd335e93233de553801f and may stay there.